### PR TITLE
Update Email-Communication-Readme.md

### PR DESCRIPTION
**Pull request by altmannyarden**
## Status
In Progress

## Related Issues
https://github.com/demisto/content/pull/11092

## Description
Changed the 'Configure the *service_mail* Parameter' section with the upcoming script change.
The service mail will be optional parameter (and not mandatory)

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@michalgold Doc review completed.

**Comment by altmannyarden** [comment]
@ShirleyDenkberg @kirbles19 I added another section for the service parameter and explanation of why and how to use it. I'll need a new review. Thank you!

**Comment by ShirleyDenkberg** [comment]
@michalgold Doc review completed. @altmannyarden and I spoke and she will be adding a screen capture of the popup.

File: content-repo/extra-docs/articles/Email-Communication-Readme.md [hunk 1]
```diff
@@ -177,15 +177,27 @@ To edit a layout, you must duplicate the layout and then edit the copy.
...
-You must configure the *service_mail* parameter.
->**Important:** This step is required!
+The *service_mail* parameter contains the sender's email address. This parameter is optional.
+
+The **SendEmailReply** script runs the ***reply-mail*** command and all the supported integrations send the email. 
+If the *service_mail* parameter is empty and only one mail sender integration is configured, an email will be sent from the email address configured in the integration. If several integrations are configured, an email will be sent from each integration (not for each instance).
+
+You can configure the *service_mail* parameter for the following uses:
+- Send the email from one default email address - If multiple email-sender integrations or instances are configured, you can configure the *service_mail* parameter to a default sender email address.
+- Send the emails from a different sender each time - This is particularly useful for MSSPs when the sender address changes per customer, incident type, etc. You can configure the parameter to be *mandatory*, so each time you click **send reply**, a pop-up will appear in which you will need to enter the service_mail email address.
```
Comment (michalgold) [inline content-repo/extra-docs/articles/Email-Communication-Readme.md:187]: so each time you click **send reply** in the layout,

**Review by michalgold — APPROVED** [review]

**Pull request merged**